addons/point_of_sale/static/src/app/screens/ticket_screen/order_details/orderline_details.xml

```diff
@@ -2,7 +2,12 @@
 <templates id="template" xml:space="preserve">
 
     <t t-name="point_of_sale.OrderlineDetails">
-        <li class="orderline w-100 m-0 py-2 px-3 lh-sm cursor-pointer" t-on-click="() => this.props.onClickOrderline(props.line)" t-att-class="{ selected: props.isSelected }">
+        <li class="orderline w-100 m-0 py-2 px-3 lh-sm cursor-pointer" 
+            t-on-click="() => this.props.onClickOrderline(props.line)" 
+            t-att-class="{ 
+                            'selected': props.isSelected,
+                            'ps-3': this.props.line.isPartOfCombo()
+                        }">
```

Comment:
What about having a component for a combo? It will be composed of orderlines with the parent and its children.

Reply:
What would be the advantage of that ?

Reply:
Just thinking out loud, I guess. With it, there is a better control on showing orderlines that belong to a group.